### PR TITLE
Quote Block Changes

### PR DESCRIPTION
**Pull request by jindaliiita**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fixes

Test URLs:
- Original: https://www.sunstar.com/healthy-thinking/cpc-containing-mouthwash-reduces-sars-cov-2-viral-load-in-vitro
- Before: https://main--sunstar--hlxsites.hlx.live/healthy-thinking/cpc-containing-mouthwash-reduces-sars-cov-2-viral-load-in-vitro
- After: https://quote-block-changes--sunstar--hlxsites.hlx.live/healthy-thinking/cpc-containing-mouthwash-reduces-sars-cov-2-viral-load-in-vitro

- [ ]  **If you are adding a new block or a variation to an existing block**, please fill below:

  Block library path: https://<branch>--sunstar--hlxsites.hlx.page/tools/sidekick/library.html?plugin=blocks&path=/sidekick/blocks/<your-block>&index=0

### CODE REVIEW

**Comment by JiangLong2019** [comment]
Looks good to me.
If you've implemented this, please ignore it. I just found there is another look & feel of quite content(lines on the top and bottom), shall we keep the same? Or make them a common one.
image

**Comment by jindaliiita** [comment]
> Looks good to me. If you've implemented this, please ignore it. I just found there is another look & feel of quite content(lines on the top and bottom), shall we keep the same? Or make them a common one. image

I have intentionally kept it same, if @ehrro feels it should be changed. I will create different variant for this.

File: blocks/quote/quote.js [hunk 1]
```diff
@@ -5,11 +5,7 @@ export default function decorate(block) {
     block.innerHTML = '';
 
     [...grandChildren].forEach((grandChild, index) => {
-      if (index === 0 && grandChild.children.length === 0) {
-        const p = document.createElement('p');
-        p.textContent = grandChild.textContent;
-        grandChild.innerHTML = '';
-        grandChild.appendChild(p);
+      if (index === 0) {
```
Comment (sdmcraft) [inline blocks/quote/quote.js:8]: @jindaliiita as you added the original changes, I assume you'd know best if removing this logic is ok.

Comment (jindaliiita) [reply blocks/quote/quote.js:8]: Yes, i have checked draft pages which i have created at that time, its working fine there.

Comment (jindaliiita) [reply blocks/quote/quote.js:8]: And Visual tests also reported no difference

Comment (sdmcraft) [reply blocks/quote/quote.js:8]: ok, I am a bit confused about why visual tests did not not report any difference. I do see difference between  
https://main--sunstar--hlxsites.hlx.page/sidekick/blocks/quote
https://quote-block-changes--sunstar--hlxsites.hlx.page/sidekick/blocks/quote

**Comment by sdmcraft** [comment]
> 🔸 3 visual differences detected
Could you sync your changes with `main` to get rid of these?

File: blocks/quote/quote.js (hunk 1 above)
Comment (jindaliiita) [reply blocks/quote/quote.js:8]: Change is intentional here but not sure why it has not been reported by the tool.

**Review by sdmcraft — APPROVED** [review]

**Comment by jindaliiita** [comment]
> > Looks good to me. If you've implemented this, please ignore it. I just found there is another look & feel of quite content(lines on the top and bottom), shall we keep the same? Or make them a common one. image
> 
> I have intentionally kept it same, if @ehrro feels it should be changed. I will create different variant for this.

@ehrro Although I have merged the PR but can you please have a look at the [url](https://quote-block-changes--sunstar--hlxsites.hlx.page/healthy-thinking/cpc-containing-mouthwash-reduces-sars-cov-2-viral-load-in-vitro). I have maintained consistency for quote block everywhere, if you feel quote block should have one more variant **"vertical"**. I will add one for that

**Pull request merged**